sdk/core/azure-core/src/main/java/com/azure/core/util/polling/implementation/OperationRequirePollContext.java

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util.polling.implementation;
+
+/**
+ * INTERNAL CLASS.
+ *
+ * An exception indicating that an operation on long-running operation
+ * requires a {@link com.azure.core.util.polling.PollingContext}
+ */
+public class OperationRequirePollContext extends RuntimeException {
```

Comment:
`PollContextRequiredException`? Prefer to suffix exceptions with `*Exception`

Reply:
yes, will do